lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -241,6 +241,19 @@ sub machine {
     return $self->settings_hash->{MACHINE};
 }
 
+sub obs_url() {
+    return "https://build.opensuse.org/";
+}
```

Comment:
In some place it needs to know where it gets its isos... for a proper version we would need to pass along that bit of information. getting it from a config file would also be good.